apps/classroomio-com/src/app.html

```diff
@@ -50,7 +50,7 @@
     <!-- <link rel="stylesheet" href="global.css"> -->
     <link rel="manifest" href="/manifest.json" crossorigin="use-credentials" />
     <link rel="icon" type="image/png" href="/favicon.ico" />
-    <link rel="icon" type="image/png" sizes="32x32" href="/logo-32.png" />
+    <link rel="icon" type="image/svg" sizes="32x32" href="/new-logo.svg" />
```

Comment:
@Chifez Please maintain the same name. You just need to update the content but keep the file name.